cmd/state-installer/internal/installer/installation_darwin.go

```diff
@@ -2,7 +2,66 @@
 
 package installer
 
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/appinfo"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/gobuffalo/packr"
+)
+
+const (
+	appDir  = "/Applications"
+	appName = "ActiveState Desktop.app"
+)
+
 // InstallSystemFiles installs files in the /Application directory
 func InstallSystemFiles(installDir string) error {
+	err := os.RemoveAll(filepath.Join(appDir, appName))
+	if err != nil {
+		return errs.Wrap(err, "Could not remove old app directory")
+	}
+
+	box := packr.NewBox("../../../../assets/macOS/app")
```

Comment:
Mike and I discussed that a little.  I initially liked the idea to *only* ship the executables in the archive.  On the other hand, we already noticed that we'll need a customary rollback mechanism for this.

So, thinking this through again, it is probably better if we stick with one system as @Naatan suggested.

Reply:
Ultimately I'll defer to you on this @mdrohmann since you're owning the update process and I can see this significantly delaying the PR. Could potentially be a later iteration also. But yeah I do think it would be better if this wasn't embedded in the binary.